modules/module-postgres-storage/README.md

```diff
@@ -0,0 +1,88 @@
+# Postgres Sync Bucket Storage
+
+This module provides a `BucketStorageProvider` which uses a Postgres database for persistence.
+
+## Configuration
+
+Postgres storage can be enabled by selecting the appropriate storage `type` and providing connection details for a Postgres server.
+
+The storage connection configuration extends the configuration for a Postgres replication source, thus it accepts and supports the same configurations fields.
+
+A sample YAML configuration could look like
+
+```yaml
+replication:
+  # Specify database connection details
+  # Note only 1 connection is currently supported
+  # Multiple connection support is on the roadmap
+  connections:
+    - type: postgresql
+      uri: !env PS_DATA_SOURCE_URI
+
+# Connection settings for sync bucket storage
+storage:
+  type: postgresql
+  # This accepts the same parameters as a Postgres replication source connection
+  uri: !env PS_STORAGE_SOURCE_URI
+```
+
+**IMPORTANT**:
+A separate Postgres server is currently required for replication connections (if using Postgres for replication) and storage. Using the same server might cause unexpected results.
```

Comment:
A separate server or separate database?

Is this just because we haven't done enough testing, or are there actual issues when using the same server?

Reply:
A separate server is required. Using different databases on the same server still results in endless keepalive events.